python/run-tests.py

```diff
@@ -161,7 +161,7 @@ def run_individual_python_test(target_dir, test_name, pyspark_python):
 
 
 def get_default_python_executables():
-    python_execs = [x for x in ["python3.6", "python2.7", "pypy"] if which(x)]
+    python_execs = [x for x in ["python3.6", "python2.7", "pypy3"] if which(x)]
```

Comment:
Does this work in Jenkins?

Reply:
BTW, could you mention this migration from `pypy` to `pypy3`?

Reply:
Ah, yeah. It wouldn't. I will fix it tomorrow.